.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -105,6 +100,28 @@ jobs:
       BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE_ARM64 }}
     secrets: inherit
 
+  publish-build-badge:
+    needs: [metadata, amd64, arm64]
+    uses: ./.github/workflows/_publish_badge.yaml
+    if: always()
```

Comment:
Instead of `always()`, use `!cancelled()` to prevent the step from running even if the workflow is cancelled.

Reply:
In this case, I think we want `always()`. If upstream workflows fail, then this workflow is canceled, but if the badge creation only happens if `!cancelled()`, then the badge does not get updated, and we see yesterday's badge.